### PR TITLE
bin: dump rocksdb statistics

### PR DESCRIPTION
**Pull request by siddontang**
Send SIGUSR1 to dump RocksDB statistics for debugging. 

@ngaut @BusyJay @zhangjinpeng1987

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -793,6 +799,16 @@ fn handle_signal(ch: SendCh<Msg>) {
...
                 encoder.encode(&metric_familys, &mut buffer).unwrap();
                 info!("{}", String::from_utf8(buffer).unwrap());
+
+                // Log common rocksdb stats.
+                if let Some(v) = engine.get_property_value(ROCKSDB_STATS_KEY) {
+                    info!("{}", v)
+                }
+
+                // Log more stats if enable_statistics is true.
+                if let Some(v) = engine.get_statistics() {
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: What's the difference between this and L804? Is it ok to just keep one?

Comment (siddontang) [inline src/bin/tikv-server.rs]: They are two different statistics in rocksdb.

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**